Xamarin.Forms.Core/Registrar.cs

```diff
@@ -95,7 +95,7 @@ internal TRegistrable GetHandler(Type type, object source, IVisual visual, param
 			return GetHandler(type) as TOut;
 		}
 
-		public TOut GetHandler<TOut>(Type type, params object[] args) where TOut : class, TRegistrable
+		public TOut GetHandler<TOut>(Type type, params TypedParameter[] args) where TOut : class, TRegistrable
```

Comment:
Since this is internal this is probably fine but we might still want to keep the old one.

Reply:
I'm fine either way on this... I didn't provide an obsolete overload here because of the internal nature.